library/common-library/templates/_labels.tpl

```diff
@@ -10,27 +10,31 @@ This function allows easily to add more labels to the function "common.labels"
 This will render the labels that should be used in all the manifests used by the helm chart.
 */}}
 {{- define "common.labels" -}}
-helm.sh/chart: {{ include "common.naming.chart" . }}
-{{ include "common.labels.selectorLabels" . }}
-{{- if .Chart.AppVersion }}
-app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
-{{- end }}
-app.kubernetes.io/managed-by: {{ .Release.Service }}
-
 {{- $global := index .Values "global" | default dict -}}
 
-{{- with $global.labels }}
-{{ toYaml . }}
-{{- end }}
+{{- $labels := dict "helm.sh/chart" (include "common.naming.chart" . ) -}}
```

Comment:
NIT: would make sense to have something like this, avoiding a merge at line 19??
```suggestion
{{- $labels := dict "helm.sh/chart" (include "common.naming.chart" . ) "app.kubernetes.io/managed-by"  .Release.Service -}}
```

Reply:
yeah, also this one works 👍

Reply:
Solved in [#fd03c2a](https://github.com/newrelic/helm-charts/pull/767/commits/fd03c2a38e80ab629ed785d602d0be35479f7823)